### DISCUSSION TITLE
Application rejected

### DISCUSSION THREAD

**Discussion by antonicotra** [community/community#130656] [GitHub Education] [answered]
### Hi everyone,

Sure, here is the translated text:

Hello everyone, I am a student of Web & Mobile Development at an academy in Italy.

I submitted the application for Github Educational, but it was rejected because the account is new (I created it about 4 days ago).

Could anyone tell me how many days of registration I need to be able to enroll?

**Comment by YuvrajxChopra** [comment, marked as answer]
I don't there is any specific account age requirement. 
But what you can do is try to create some repositories, update those repositories, follow some accounts, fork repositories you know just do some activities, also you can verify your email address and add your info in your account. And I think in few weeks after this you will be good.
